client/src/Pos/Client/Txp/Network.hs

```diff
@@ -69,6 +70,18 @@ prepareMTx hdwSigners pendingAddrs inputSelectionPolicy addrs outputs addrData =
     utxo <- getOwnUtxos (toList addrs)
     eitherToThrow =<< createMTx pendingAddrs inputSelectionPolicy utxo hdwSigners outputs addrData
 
+-- | Construct unsigned Tx
+prepareUnsignedTx
+    :: TxMode m
+    => PendingAddresses
+    -> InputSelectionPolicy
+    -> NonEmpty Address
+    -> NonEmpty TxOutAux
+    -> m (Tx, NonEmpty TxOut)
+prepareUnsignedTx pendingAddrs inputSelectionPolicy addrs outputs = do
+    utxo <- getOwnUtxos (toList addrs)
+    eitherToThrow =<< createUnsignedTx pendingAddrs inputSelectionPolicy utxo outputs
+
```

Comment:
This is for CHW-84, we're preparing raw, unsigned transaction, which will be signed externally (for example, in Ledger device).